### PR TITLE
Migrate Data Feeds to mdx

### PR DESCRIPTION
**Pull request by thedriftofwords**
Migrate Data Feeds files to mdx

### CODE REVIEW

File: src/pages/data-feeds/price-feeds/addresses.mdx [hunk 1]
```diff
@@ -6,8 +6,8 @@ metadata:
   title: "Price Feed Contract Addresses"
   description: "A list of Price Feed addresses on supported networks."
 date: Last Modified
-setup: |
-  import FeedPage from "@features/feeds/components/FeedPage.astro"
 ---
 
-<FeedPage  />
+import FeedPage from "@features/feeds/components/FeedPage.astro"
```
Comment (dwightjl) [inline src/pages/data-feeds/price-feeds/addresses.mdx:11]: Somehow a slightly different CSS is getting applied in the MDX version. The network selectors are bunched close together and need more padding.

Preview: https://documentation-git-fork-thedriftofwords-mdx-a2471c-chainlinklabs.vercel.app/data-feeds/price-feeds/addresses

Current page: https://docs.chain.link/data-feeds/price-feeds/addresses

image

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: src/pages/data-feeds/price-feeds/addresses.mdx (hunk 1 above)
Comment (thedriftofwords) [reply src/pages/data-feeds/price-feeds/addresses.mdx:11]: It appears normal in my local environment, where I still have Astro 2.0.2 installed. So I expect this problem to go away when we update the package.json and configs. I can revert this specific file for now and then add it to PR #1189.

Comment (thedriftofwords) [reply src/pages/data-feeds/price-feeds/addresses.mdx:11]: Reminder: 

- The same issue also affects the Proof of Reserve addresses page (https://docs.chain.link/data-feeds/proof-of-reserve/addresses). 
- The NFT Floor Price one (https://docs.chain.link/data-feeds/nft-floor-price/addresses) only has one network selector right now.

**Comment by thedriftofwords** [comment]
Weird things that happened in Astro 2.0.2 local env with price feeds addresses pages:

- Entire page content (except title) disappears when reverting the files back to .md
- The padding / gap weirdness doesn't exist

File: src/pages/data-feeds/price-feeds/addresses.mdx (hunk 1 above)
Comment (thedriftofwords) [reply src/pages/data-feeds/price-feeds/addresses.mdx]: I've handled this by reverting the addresses files to .md. We can change them back to .mdx as part of #1189

File: src/pages/data-feeds/solana/using-data-feeds-off-chain.mdx [hunk 2]
```diff
@@ -37,49 +37,49 @@ You can [add data feeds to an existing off-chain project](#adding-data-feeds-to-
...
 
-1. Install the latest Mainnet version of [the Solana CLI](https://github.com/solana-labs/solana/releases) and export the path to the CLI:
+1.  Install the latest Mainnet version of [the Solana CLI](https://github.com/solana-labs/solana/releases) and export the path to the CLI:
 
-   ```shell
-   sh -c "$(curl -sSfL https://release.solana.com/v1.13.6/install)" &&
-   export PATH="~/.local/share/solana/install/active_release/bin:$PATH"
-   ```
+    ```shell
+    sh -c "$(curl -sSfL https://release.solana.com/v1.9.28/install)" &&
```
Comment (dwightjl) [inline src/pages/data-feeds/solana/using-data-feeds-off-chain.mdx:51]:
```suggestion
    sh -c "$(curl -sSfL https://release.solana.com/v1.13.6/install)" &&
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: src/features/feeds/shared/using-price-feeds.mdx [hunk 3]
```diff
@@ -24,18 +29,20 @@ The contract has the following components:
...
 
 Deploy the `PriceConsumerV3` smart contract on the Sepolia testnet.
 
 1. [Open the example contract](https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol) in Remix. Remix opens and shows the contents of the smart contract.
 
    <div class="remix-callout">
-   <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol">Open the contract in Remix</a>
+     <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol">
+       Open the contract in Remix
+     </a>
```
Comment (dwightjl) [inline src/features/feeds/shared/using-price-feeds.mdx:45]:
```suggestion
     <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol" target="_blank">
       Open the contract in Remix
     </a>
```

This one has the CSS issue again. I think you said `target="_blank"` fixes it?

**Review by dwightjl — APPROVED** [review]

File: src/features/feeds/shared/using-price-feeds.mdx (hunk 3 above)
Comment (thedriftofwords) [reply src/features/feeds/shared/using-price-feeds.mdx]: I thought _removing_ the `target` attribute fixed this, but now I think it's actually forcing the link to go within one line in the source code. https://github.com/smartcontractkit/documentation/pull/1206/commits/be8cb3a136ffd6cb75a9a4167b624e3f223246db

The issue doesn't make sense to me in the first place, so I don't know why this fixes it, but it seems to work.

File: src/pages/data-feeds/solana/using-data-feeds-off-chain.mdx (hunk 2 above)
Comment (thedriftofwords) [reply src/pages/data-feeds/solana/using-data-feeds-off-chain.mdx]: Applied.

**Pull request merged**